### ISSUE TITLE
add markdownLint as an npm package

### ISSUE THREAD

**Issue by danisyellis** [indeedeng/starfish#124]
Some devs working on Starfish use the VSCode markdownLint package and it was confusing that they were getting tons of linting errors. So I 
1. fixed some of the linting errors in README.md and CONTRIBUTING.MD
2. added a few  comments into the README

To make things more consistent, it would probably be best if the project had markdown linting included in it via an npm package  for a markdownLint command-line interface.
I believe these are the steps:
1. Decide whether to use https://www.npmjs.com/package/markdownlint-cli or https://www.npmjs.com/package/markdownlint-cli2 
      - The second one comes from the same dev who made the library and VS Code extension, so my first inclination would be to use that one. But the first one has a lot more downloads. So a little bit of research needs to be done before deciding. One way to decide could be using the one whose documentation makes more sense to you.
2. add it to Starfish as a dev dependency i.e. ( `npm i -D markdownlint-cli2` )
3.  I think the next step would be to create a markdownLint configuration file that turns off the rules that are currently being disabled in the README. 
4. make sure it still lints cleanly

**Comment by IngridGdesigns** [comment]
which is the markdown VScode plugin that was used, i want to reproduce the errors, is it by David Anson?

**Comment by danisyellis** [comment]
Yup, it's this one https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
